React/CalcApp/e2e/project.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
is this necessary? we will not deal with e2e so i do not want to create confusion

Reply:
I removed the e2e folder with all its content